### ISSUE TITLE
Update the Kaggle description

### ISSUE THREAD

**Issue by m-kruth** [2kruman/COVID19-NZ-known-cases#4]
- Add references for more complex concepts.
- Make it easily readable.
- Make it clear where help is needed.

**Comment by RapidCompiler** [comment]
Could you please elaborate on this issue?

**Comment by m-kruth** [comment]
I've changed my mind on this issue: I think the description is readable, it just needs to be updated (link here: https://www.kaggle.com/madhavkru/covid19-nz). I'll add in a section in the description for the limitations of that dataset that doc that you wrote up after I add my thoughts too. I'll also add links to definitions for any statistical terms that I've used. Not everyone who looks at this dataset will have a background in statistics so I think that it would be good to use simple language and clarify _any_ statistical term used.
I also want to add in a glossary section describing each column, especially with the excellent detective work you did figuring out the MOH's definitions for `infectious period` and `location`.
